libsoftwarecontainer/src/gateway/networkgateway.cpp

```diff
@@ -347,3 +333,54 @@ bool NetworkGateway::isBridgeAvailable()
     return isSuccess(m_netlinkHost.hasAddress(addresses, AF_INET, m_gateway.c_str()));
 }
 
+std::string IPTableEntry::getChain() {
+    if ( "INCOMING" == m_type) {
+        return "INPUT";
+    } else if ("OUTGOING" == m_type) {
+        return "OUTPUT";
+    } else if ("FORWARD" == m_type) {
+        return "FORWARD";
+    }
+    return "";
```

Comment:
I agree this is not good solution. Imho m_type should be already compatible with iptables chai. So i suggest to change configuration value from 
INCOMING to INPUT
OUTGOING to OUTPUT. 

If it is done so, this function wouldn't be even needed.